CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- Upgrade go-version to v1.23.5
```

Comment:
As far as I understood the reason to upgrade the go compiler was to fix CVEs? Shouldn't this be reflected in the change log. And does changing the compiler introduce a change so strong it should be mentioned in the changelog?

And since we have the issue #36 at hand, it should be mentioned here